domains/devanshukothe.json

```
@@ -0,0 +1,18 @@
+
+{
+
+  "owner": {
+
+    "username": "Devanshu_kothe",
```

change to your username
```suggestion
    "username": "Devanshukothe",
```